client/landing/stepper/hooks/use-translate-flow-name.ts

```diff
@@ -0,0 +1,19 @@
+import { useTranslate } from 'i18n-calypso';
+
+export function useTranslateFlowName( flow: string | null ): string {
+	const translate = useTranslate();
+
+	switch ( flow ) {
+		case 'newsletter':
+			return translate( 'Newsletter' );
+			break;
+		case 'link-in-bio':
```

Comment:
We might want to just translate the entire sentences here. like `translate( 'Your Newsletter is up and running' )`.

I would tend to overtranslate and not be so DRY, as a lot of languages work differently. While it may have the same result in some languages like english, trying to piece translations together like this could have unintended translations in other languages.

Reply:
This has been done!